ui/i18n/src/main/res/values/strings.xml

```diff
@@ -125,6 +125,9 @@
     <string name="feed_auto_download_global">Global default</string>
     <string name="feed_auto_download_always">Always</string>
     <string name="feed_auto_download_never">Never</string>
+    <string name="feed_new_episodes_action_global">Global default</string>
```

Comment:
Now we have two strings saying "Global default". That could lead to inconsistencies. I would rename the existing one to something more generic and use the same string in both locations.

Reply:
Done.